packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1425,6 +1425,15 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     }
   }
 
+  void _closeConnectionReceived() {
```

Comment:
It would probably be good to document the difference between this and the previous method for future readers.

Reply:
I removed this method. Only "connectionClosed" remains now which is documented on the interface.